addons/core/functions/events/fnc_onSplendidCamera.sqf

```diff
@@ -0,0 +1,34 @@
+#include "script_component.hpp"
+
+/*
+  Name: TFAR_fnc_onSplendidCamera
+
+  Author: Ampersand
+    Eventhandler that fires when opening the Splendid Camera
+
+  Arguments: - Passed from Extended_DisplayUnload_EventHandlers >> RscDisplayCamera
+    0: Display <DISPLAY>
+    1: Event Type <NUMBER>
+
+  Return Value:
+    None
+
+  Example:
+    [] call TFAR_fnc_onSplendidCamera;
+
+  Public: No
+*/
+params [["_display",displayNull], ["_eventType","Open"]];
+
+switch _eventType do {
```

Comment:
I intended it to be for Curators and Video-makers to have finer camera control while hearing local voice, and not as a Spectator mode, which I'm don't have much experience with.

`releaseAllTangents` definitely should be included.
Hiding the Splendid Cam HUD using Backspace seems to hide everything, as I haven't noticed TFAR HUD elements.
Hearing all other spectators did not occur to me at all since it's not included in the intended use. I have no preference either way.

Currently I can't talk while in Splendid Cam and haven't figured out how yet. that would definitely be desirable.
Hearing enemies at location is desirable.
Hearing intercom is actually desirable for certain recording effects.
I had not considered "mixing" sound from camera and body positions. I don't think that's necessary, but if it could be done easily, may be a way to allow players to get the attention of the person in Splendid Cam.